### PR TITLE
Email Forward Management: Update support documentation

### PR DESCRIPTION
**Pull request by kojdm**
Updates docs with changes related to email forward management

| Current | Updated |
|--------|--------|
| https://support.dnsimple.com/articles/cancel-subscription | https://deploy-preview-1299--dnsimple-support.netlify.app/articles/cancel-subscription |
| https://support.dnsimple.com/articles/reactivate-subscription | https://deploy-preview-1299--dnsimple-support.netlify.app/articles/reactivate-subscription |
| https://support.dnsimple.com/articles/email-forwarding/ | https://deploy-preview-1299--dnsimple-support.netlify.app/articles/email-forwarding |
| https://support.dnsimple.com/articles/what-happens-if-i-stop-paying/ | https://deploy-preview-1299--dnsimple-support.netlify.app/articles/what-happens-if-i-stop-paying | 

Closes https://github.com/dnsimple/dnsimple-app/issues/28995
Belongs to https://github.com/dnsimple/dnsimple-business/issues/2061

### CODE REVIEW

**Review by ggalmazor — APPROVED** [review]

File: content/articles/reactivate-subscription.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ categories:
 
 When you unsubscribe, or your account was [closed due to non-payment](/articles/account-suspended/), all of your domains and records remain in your account. You can reactivate your subscription and start using DNSimple's services again at any time.
 
-When your account is deactivated, [WHOIS privacy](/articles/whois-privacy/) and [domain auto-renewal](/articles/domain-auto-renewal/) are immediately disabled. If you want to use these features, you'll need to manually re-activate them.
+When your account is deactivated, [WHOIS privacy](/articles/whois-privacy/), [domain auto-renewal](/articles/domain-auto-renewal/), and [email forwarding](/articles/email-forwarding/) are immediately disabled. If you want to use these features, you'll need to manually re-activate them.
```
Comment (Pemacaa) [inline content/articles/reactivate-subscription.md:12]: This  part here is incorrect:

> If you want to use these features, you'll need to manually re-activate them.

When an account is resubscribed, it retains the previous preferences., which means that any domains with auto-renewal enabled will have auto-renewal reactivated automatically, and the same applies to email forwarding settings.

**Review by Pemacaa — APPROVED** [review]

File: content/articles/reactivate-subscription.md (hunk 1 above)
Comment (itsalyse) [reply content/articles/reactivate-subscription.md:12]: @Pemacaa could we say this? 

> If you reactivate your account, all previous preferences will be reactivated as well, including WHOIS privacy, auto-renewal, and email forwarding.

Comment (stephanoodle) [reply content/articles/reactivate-subscription.md:12]: @itsalyse that works for me.

**Review by stephanoodle — APPROVED** [review]

File: content/articles/reactivate-subscription.md (hunk 1 above)
Comment (kojdm) [reply content/articles/reactivate-subscription.md]: Thank you for handling this!

**Pull request merged**